trappy/plotter/AbstractDataPlotter.py

```diff
@@ -73,16 +74,22 @@ def _parse_value(self, signal_def):
         :type signal_def: str
         """
 
-        event, column = signal_def.split(":")
+        match = re.match("(?P<event>[^:]+):(?P<column>[^:]+)(:.+)?", signal_def)
```

Comment:
regarding the first suggestion, I have added the group name but dropped the check on color.  With it we can warn the user if they make a mistake.  For example: thermal:temp:23,43.0 silently ignores the color.  I prefer to capture it in the regexp and then barf if we can't parse it.